src/shipit_static_analysis/shipit_static_analysis/lint.py

```diff
@@ -155,14 +154,14 @@ def find_issues(self, path):
             logger.warn('Invalid json output', path=path, lines=lines)
             raise
 
-        full_path = os.path.join(self.repo_dir, path)
+        full_path = os.path.join(settings.repo_dir, path)
         if full_path not in payload and path not in payload:
             logger.warn('Missing path in linter output', path=path)
             return
 
         # Mozlint uses both full & relative path to index issues
         return [
-            MozLintIssue(self.repo_dir, path, **issue)
+            MozLintIssue(path, **issue)
```

Comment:
Should `path` be `p`?

Reply:
Yeap, same bad rebase

Reply:
Actually, no, it must be `path` as MozlintIssue needs the relative path.

Reply:
Right, this code is a bit confusing, maybe because of the two for loops in the comprehension. It might be worth refactoring it a bit, but we can do it later on.

Reply:
:+1: